deps/wazuh_testing/wazuh_testing/dashboard_saturation_tests/dashboard_saturation_tests.py

```diff
@@ -0,0 +1,217 @@
+import argparse
+import json
+import pandas as pd
+
+from datetime import datetime
+from os import makedirs
+from os.path import exists
+from subprocess import run
+
+# Global Configuration Variables
+csv_types = ['aggregate', 'intermediate']
+url_format = 'https://'
```

Comment:
Is this used exclusively in the `gen_url` function? If so, consider relocating it there.

Reply:
Solved in https://github.com/wazuh/wazuh-qa/pull/5629/commits/ca020ab5fd076f3ff705605233b10f1ca0921806